### ISSUE TITLE
ignores {exp:channel:entries status="closed"}

### ISSUE THREAD

**Issue by benya** [ExpressionEngine/ExpressionEngine#841]
ЕЕ 6.1 ignored request {exp:channel:entries channel="news" limit="25" status="closed"}
although at 54 worked

**Comment by intoeetive** [comment]
@benya what exactly has been "ignored"? Please provide us sufficient information so we could replicate the bug

**Comment by benya** [comment]
For this request, I will be shown material with the status "closed"

After I updated ЕЕ 5.4 -> ЕЕ 6.x this request stopped working.

**Comment by intoeetive** [comment]
The minimal code
`{exp:channel:entries channel="blog" limit="25" status="closed"}{title}{/exp:channel:entries}`
is working fine on my installation of 6.0.1

If you are having issue with certain custom field please provide all the information

**Comment by benya** [comment]
I tried to record the problem on video and uploaded the video to YouTube

https://youtu.be/ZByylKXvXXo

**Comment by intoeetive** [comment]
It's a little bit hard to follow as your template and UI has some other code/elements. I can see you don't get expected results, but did you check the page generated HTML source - maybe that's UI issue? 
If you're absolutely positive you can replicate the issue with JUST this code `{exp:channel:entries channel="job" limit="25" status="closed"}{title}{/exp:channel:entries}` and no extra HTML, try temporary disabling extensions via config file and see if that helps.

**Comment by benya** [comment]
I created a new template for the test and added only this code there

`{exp:channel:entries channel="job" limit="25" status="closed"}{title}{/exp:channel:entries}`

As I said, this request worked for me on version 5.4, but when I upgraded to version 6.1, this request stopped working for me, namely, the status="closed" function stopped working

https://youtu.be/BwmAuJbqnnE

**Comment by intoeetive** [comment]
Could it be that closed entries have been expired? I have seen on your screencast you have bunch of closed entries, but they might be hidden because they are expired.
Does it work if you specify `status="open|closed"`?
Can you experiment with certain entry by changing its status?
Lastly, did you check with extensions disabled?

**Comment by benya** [comment]
Indeed, the problem was that the records had expired.

This is how it works :)

`{exp:channel:entries channel="job" limit="25" show_expired="yes" status="closed"}`

It really was my fault.

Sorry for wasting time.
